### PR TITLE
[MIG] project_task_code: Migrate to 10.0

### PR DESCRIPTION
**Pull request by kellylougheed**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Why don't you start from the 9.0 version using migration guide: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-10.0?

**Comment by kellylougheed** [comment]
Will do @pedrobaeza -- thanks!

**Comment by pedrobaeza** [comment]
No need to close this. You can just push force with the same branch name if you want

**Comment by lasley** [comment]
Hah yeah it was an auto-close after a reset then a failed format-patch & a subsequent push -f, we're working on the git wizardry right now 👍

**Comment by lasley** [comment]
Awwwww but it didn't reopen when the branch was pushed to again. Shame Github, Shame! 

@kellylougheed can you reopen from your end?

**Comment by dreispt** [comment]
AFAICT you're basically proposing for v10 the changeset merged on v9, correct?

**Comment by kellylougheed** [comment]
@dreispt Yes, this should just be an upgrade to v10

File: project_task_code/__manifest__.py [hunk 1]
```diff
@@ -1,35 +1,22 @@
...
-#
-##############################################################################
+# -*- coding: utf-8 -*-
+# Copyright 2016 Tecnativa <vicent.cubells@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
     "name": "Sequential Code for Tasks",
-    "version": "8.0.1.0.0",
+    "version": "9.0.1.0.0",
```
Comment (dreispt) [inline project_task_code/__manifest__.py]: 10.0.1.0.0

File: project_task_code/README.rst [hunk 2]
```diff
@@ -1,19 +1,43 @@
...
+To use this module, you need to:
+
+#. Go to menu Project > Search > Tasks and create a new task, and you get a
+new code saving it.
+#. If you duplicate a task, you will get a new code for the new task.
+
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/140/9.0
```
Comment (dreispt) [inline project_task_code/README.rst]: s/9/10

File: project_task_code/README.rst [hunk 3]
```diff
@@ -1,19 +1,43 @@
...
 
 Bug Tracker
 ===========
 
-Bugs are tracked on `GitHub Issues <https://github.com/OCA/project-service/issues>`_.
-In case of trouble, please check there if your issue has already been reported.
-If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
-`here <https://github.com/OCA/project-service/issues/new?body=module:%20project_task_code%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/project-service/issues>`_. In case of trouble, please
```
Comment (dreispt) [inline project_task_code/README.rst]: s/project-service/project

**Review by dreispt — CHANGES_REQUESTED** [review]
LGTM expect for some small nitpicks

File: project_task_code/views/project_view.xml [hunk 4]
```diff
@@ -8,7 +8,7 @@
             <field name="arch" type="xml">
                 <field name="name" position="before">
                     <field name="code" class="oe_inline" />
-                    <label string=" - " class="oe_inline" />
+                    <label for=" - " class="oe_inline" />
```
Comment (dreispt) [inline project_task_code/views/project_view.xml]: Are you sure about this one? AFAICT `string` was correct.

Comment (kellylougheed) [reply project_task_code/views/project_view.xml]: I was going off of this document, which said to change XML attributes called "string": https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-10.0 I can change it back, though.

Comment (lasley) [reply project_task_code/views/project_view.xml]: It's referring to string selectors, which means an [Xpath](http://www.odoo.yenthevg.com/xpath-expressions-in-odoo-8/) targetting a string attribute such as this one. The string attributes themselves aren't bad, just the use of the string as an Xpath injection point when inheriting.

Along those lines it is good to add name attributes to important objects like notebook pages that only have strings in order to allow for easier targeting. Not applicable here though 😄

Comment (kellylougheed) [reply project_task_code/views/project_view.xml]: OK, got it. I'll change it back!

**Review by lasley — APPROVED** [review]
Thanks @kellylougheed

**Review by pedrobaeza — APPROVED** [review]
Squashing before merging

**Review by dreispt — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@dreispt, I asked for a squash before merging. Now, there are 2 useless commits "Requested README changes" and "Requested XML change" on the repo...

**Pull request merged**